src/pydata_sphinx_theme/assets/styles/base/_base.scss

```diff
@@ -178,4 +178,34 @@ pre {
   color: var(--pst-color-secondary-text);
   background-color: var(--pst-color-secondary);
   border: none;
+
+  .fa-arrow-up {
+    // Using margin instead of a space character prevents the space between the
+    // icon and the text from being underlined when the button is hovered.
+    margin-inline-end: 0.5em;
+  }
```

Comment:
slight hesitation about adding properties to fontawesome glyphs in this way, in case it affects users' ability to use those glyphs within their docs (especially for something as generic as an up-pointing arrow). Can we add a `pst-*` class to the arrow where it appears in the "back to top" button, and attach `margin-inline-end` to that class instead of directly to the fontawesome class?